sqrl-tools/sqrl-run/src/main/java/com/datasqrl/DatasqrlRun.java

```diff
@@ -112,33 +138,65 @@ public CompiledPlan compileFlink() {
     config.putIfAbsent("table.exec.source.idle-timeout", "1 s");
     config.putIfAbsent("taskmanager.network.memory.max", "800m");
     config.putIfAbsent("execution.checkpointing.interval", "30 sec");
+    config.putIfAbsent("execution.checkpointing.min-pause", "20 s");
     config.putIfAbsent("state.backend", "rocksdb");
     config.putIfAbsent("table.exec.resource.default-parallelism", "1");
+    config.putIfAbsent("execution.target", "remote");
+    config.putIfAbsent("rest.address", "localhost");
 
     Configuration configuration = Configuration.fromMap(config);
-    StreamExecutionEnvironment sEnv = StreamExecutionEnvironment.createLocalEnvironmentWithWebUI(configuration);
+    String udfJarPath = "/Users/henneberger/sqrl/sqrl-testing/sqrl-integration-tests/src/test/resources/udf/build/deploy/flink/lib/myjavafunction-0.1.0-snapshot-all.jar";
+//    String sqrlRunJarPath = "/Users/henneberger/sqrl/sqrl-tools/sqrl-run/target/sqrl-run.jar";
```

Comment:
Same deal as udf jars, will come from ${SYSTEM_JAR_DIR}, which will be added by a build gradle step. This can be a compiler flag so we can build for different flink deployment scenarios